lib/MySQL_Session.cpp

```diff
@@ -22,8 +22,18 @@
 
 #define SELECT_VERSION_COMMENT "select @@version_comment limit 1"
 #define SELECT_VERSION_COMMENT_LEN 32
+
 #define PROXYSQL_VERSION_COMMENT "\x01\x00\x00\x01\x01\x27\x00\x00\x02\x03\x64\x65\x66\x00\x00\x00\x11\x40\x40\x76\x65\x72\x73\x69\x6f\x6e\x5f\x63\x6f\x6d\x6d\x65\x6e\x74\x00\x0c\x21\x00\x18\x00\x00\x00\xfd\x00\x00\x1f\x00\x00\x05\x00\x00\x03\xfe\x00\x00\x02\x00\x0b\x00\x00\x04\x0a(ProxySQL)\x05\x00\x00\x05\xfe\x00\x00\x02\x00"
 #define PROXYSQL_VERSION_COMMENT_LEN 81
+
+// PROXYSQL_VERSION_COMMENT_WITH_OK is sent instead of PROXYSQL_VERSION_COMMENT
+// if Client supports CLIENT_DEPRECATE_EOF
+#define PROXYSQL_VERSION_COMMENT_WITH_OK "\x01\x00\x00\x01\x01" \
+"\x27\x00\x00\x02\x03\x64\x65\x66\x00\x00\x00\x11\x40\x40\x76\x65\x72\x73\x69\x6f\x6e\x5f\x63\x6f\x6d\x6d\x65\x6e\x74\x00\x0c\x21\x00\x18\x00\x00\x00\xfd\x00\x00\x1f\x00\x00" \
+"\x0b\x00\x00\x03\x0a(ProxySQL)" \
+"\x07\x00\x00\x04\xfe\x00\x00\x02\x00\x00\x00"
+#define PROXYSQL_VERSION_COMMENT_WITH_OK_LEN 74
```

Comment:
This is basically the response of `select @@version_comment limit 1` which is sent as soon as you start a mysql client from command line. Since one of the EOFs does not exist anymore, it is now only 4 packets long and looks like this:
<img width="543" alt="Screen Shot 2020-04-30 at 7 45 45 PM" src="https://user-images.githubusercontent.com/51671/80769589-b1b49080-8b1b-11ea-9bb9-265bfb51932a.png">

Wireshark has a hard time reading OK packets with FE and thinks its the old EOF packets.

Compare this with how the response looked like previously. You will see 5 packets below. Also note, the EOF packet is of size 5 bytes vs the OK packet which is of size 7 above.

<img width="543" alt="Screen Shot 2020-04-30 at 7 44 33 PM" src="https://user-images.githubusercontent.com/51671/80769644-e0cb0200-8b1b-11ea-9506-07edc4835a64.png">